### DISCUSSION TITLE
可以在 option 里配置参数，然后传递给 add_requires 的参数吗

### DISCUSSION THREAD

**Discussion by Elite-stay** [xmake-io/xmake#5326] [Q&A] [answered]
除了 $(xxx) 的方式获取 option 的值之外还能怎么获取呢，我试了传递给 add_requires 不会展开成对应的值，然后就报错了

或者如果 package 里可以加 option 的话也行？

**Comment by waruqi** [comment, marked as answer]
get_config/has_config/is_config
